simplecpp.cpp

```diff
@@ -1843,46 +1843,40 @@ namespace simplecpp {
 class ScopedLock
 {
 public:
-    explicit ScopedLock(HANDLE mutex)
-        : m_mutex(mutex)
+    explicit ScopedLock(CRITICAL_SECTION& criticalSection)
+        : m_criticalSection(criticalSection)
```

Comment:
These changes were merged in another PR as far as I know... so I assume you will remove them or create a new PR..